src/DW1000Ranging.cpp

```diff
@@ -348,6 +348,8 @@ short DW1000RangingClass::detectMessageType(byte datas[]) {
 		//we have a short mac frame message (poll, range, range report, etc..)
 		return datas[SHORT_MAC_LEN];
 	}
+
+	return -1;
```

Comment:
-1 seems to be safe - but maybe you want some define for unrecognized packet type?